### DISCUSSION TITLE
What is the actual amount of insulin delivered on every loop?

### DISCUSSION THREAD

**Discussion by AFractalThought** [Artificial-Pancreas/iAPSdocs#168] [Q&A]
https://www.reddit.com/r/diabetes_t1/comments/1eb6zx5/users_of_iaps_what_is_the_actual_amount_of/

In the history of treatments section, I am assuming that a log entry(ex: 12:59 PM Temp Basal 1 U/hr 30 min) means that at 12:59 PM, insulin was delivered and that insulin is only delivered at these times (except for bolus). So how much insulin was actually delivered at 12:59 PM?

**Comment by 53645714n** [comment]
There was a temp basal set for 1u/hr for 30 minutes. (So, in the next half hour, by basal, you were given 0,5 units of insulin.) No bolus was given.
